### ISSUE TITLE
Split training data into training and validation datasets, create a V1 CNN model and train it. 

### ISSUE THREAD

**Issue by nhsjdiwlcnsv** [RAXIOM/raxiom-ml#3]
Need to create two datasets from the training data: actual training dataset and a validation one. Training dataset will be used for model training, while validation dataset will be used to track the loss value and prevent the model from overfitting.

Also, you may want to store different versions of data (the original one and the one with paths to files instead of IDs in ```BraTS21ID``` column) in different ```.csv``` files before splitting the training dataset.
